### DISCUSSION TITLE
2.9.2 haptic still happens

### DISCUSSION THREAD

**Discussion by sam2b** [EdgeTX/edgetx#4341] [Q&A]
Hello.
After flashing [2.9.2 firmware](https://github.com/EdgeTX/edgetx/pull/4285/commits/e500bae2c342d436d056f8745a053abb6001efee) and bootloader on my TX16, the haptic still happens during power-off and power-on.
Is there something I might be doing incorrectly?  Else what could be the problem?

After flashing both, I confirmed 2.9.2 is present in the system information screen.

Thanks.

**Comment by pfeerick** [comment]
Do you have the haptic set to "Quiet"?

**Reply by sam2b** [reply]
No, I do not.  There are no instructions how to utilize this feature, so is that the solution?

**Reply by pfeerick** [reply]
Yes, that is how it is currently implemented.

From the 2.9.2 release notes: 
![brave_M8gQ7WYpKp](https://github.com/EdgeTX/edgetx/assets/5500713/04b71366-342c-4939-8ed5-13f6fba43019)
